docs_app/src/app/layout/doc-viewer/doc-viewer.component.ts

```diff
@@ -241,4 +247,17 @@ export class DocViewerComponent implements OnDestroy {
         }),
     );
   }
+
+  private addDocumentMetaTags(title: string, description: string | null): void {
+    this.metaService.updateTag({ name: 'twitter:title', content: title });
+    this.metaService.updateTag({ name: 'twitter:card', content: 'summary' });
+    this.metaService.updateTag({ property: 'og:title', content: title });
```

Comment:
There are also `twitter:site` and `twitter:image` tags. What the image ought to be, I dunno - maybe the pink eel? - but the site is useful, as it's the URL.

Reply:
@cartant can you please create a separate issue to track this and add @JWO719  as the owner?